_includes/footer.html

```diff
@@ -60,7 +60,7 @@ <h4 class="footerHeaderStyle align-center-md"><b>Development</b></h4>
 			    <li class="footerListStyle"><a href="{{ site.baseurl }}/develop">Read about development</a></li>
                             <li class="footerListStyle"><a href="https://wiki.sugarlabs.org/go/Mailing_Lists">Join mailing list</a></li>
                             <li class="footerListStyle"><a href="https://help.sugarlabs.org/en/">Documentation</a></li>
-                            <li class="footerListStyle"><a href="https://matrix.to/#/#sugar:matrix.org">Chat with us on Matrix</a></li>
+                            <li class="footerListStyle"><a href="{{ site.baseurl }}/chat-with-the-community">Chat with the community</a></li>
```

Comment:
"Chat with the community on Discord or Matrix"


This will help people specifically searching those terms.